piet-gpu/shader/kernel4.comp

```diff
@@ -128,6 +130,10 @@ void main() {
         if (xy_uint.x < 1024 && xy_uint.y < 1024) {
             rgb[i] = imageLoad(images[gl_WorkGroupID.x / 64], ivec2(xy_uint + chunk_offset(i))/4).rgb;
         }
+#else
+        if (xy_uint.x < 1024 && xy_uint.y < 1024) {
+            rgb[i] = imageLoad(images[0], ivec2(xy_uint + chunk_offset(i))/4).rgb;
+        }
```

Comment:
This unconditionally copies images[0] to the output. This used to not affect me, because I'm not using ENABLE_IMAGE_INDICES. Can we replace this debug output of images with a FillImage command from Rust? Or more expedient, hide the copying behind a DEBUG flag?

Also, this needs rebasing on top of the recently merged RGBA output support.

Reply:
Moved it behind a debug.